<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>646</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="ExposureTime">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>170</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Exposure Time (&lt;span style=&quot; font-family:'sans-serif'; font-size:14px; color:#202122; background-color:#ffffff;&quot;&gt;μ&lt;/span&gt;s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SliceThickness">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Slice Thickness (μm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="UVIntensity">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;UV Intensity (0-255)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="SelectResin">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Select Resin&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetExposureTime">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>190</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetSliceThickness">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>130</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="SetUVIntensity">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>250</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="Title">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>371</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:14pt;&quot;&gt;Experimental CLIP 3D Printer GUI&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ResinSelect">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>130</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Resin A</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resin B</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Resin C</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="ResinSettlingTime">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>170</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Resin Settling Time (&lt;span style=&quot; font-family:'sans-serif'; font-size:14px; color:#202122; background-color:#ffffff;&quot;&gt;μ&lt;/span&gt;s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ExposureTimeParam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="SliceThicknessParam">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>130</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="ResinSettlingTimeParam">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>190</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetResinSettlingTime">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>190</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QListWidget" name="FileList">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>491</y>
      <width>301</width>
      <height>91</height>
     </rect>
    </property>
   </widget>
   <widget class="QToolButton" name="SelectFile">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>470</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Select File</string>
    </property>
   </widget>
   <widget class="QLabel" name="ObjectImageFiles">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>470</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Object Image Files&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QScrollBar" name="OutputScroll">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>360</y>
      <width>21</width>
      <height>221</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QScrollArea" name="ProgramOutput">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>360</y>
      <width>281</width>
      <height>221</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>279</width>
       <height>219</height>
      </rect>
     </property>
     <widget class="QTextBrowser" name="ProgramPrints">
      <property name="geometry">
       <rect>
        <x>-5</x>
        <y>-9</y>
        <width>291</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="InitializeAndSynchronize">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>190</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Initialize
 and
 Sychronize</string>
    </property>
   </widget>
   <widget class="QLabel" name="ProgramOutputLabel">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>330</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Program Ouput&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartPrint">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>260</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(0, 255, 0); border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>Start Print</string>
    </property>
   </widget>
   <widget class="QLabel" name="StageVelocity">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>230</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Stage Velocity (mm/s)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="UVIntensityParam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="StageSpeedParam">
    <property name="geometry">
     <rect>
      <x>211</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetStageVelocity">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="MinEndOfRun">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Min end of run (mm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="MaxEndOfRun_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>350</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Max end of run (mm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="MinEndOfRunParam">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetMinEndOfRun">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="MaxEndOfRun">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>370</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetMaxEndOfRun">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>370</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ManualLightEngine">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>120</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual 
Light Engine Control</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ManualStage">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>120</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Manual
 Stage Control</string>
    </property>
   </widget>
   <widget class="QSlider" name="CurrentStagePos">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>431</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentStagePosition">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>410</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Current Stage Position (mm)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QToolButton" name="ViewGraphics">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>470</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QLabel" name="CurrentPositionIndicator">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>430</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 255, 255); border: 1px solid black;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="StartPrint_2">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>260</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:rgb(255, 0, 0);border: 1px solid black;</string>
    </property>
    <property name="text">
     <string>Abort</string>
    </property>
   </widget>
   <widget class="QLabel" name="StageVelocity_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>290</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Stage Accel. (mm/s&lt;span style=&quot; vertical-align:super;&quot;&gt;2&lt;/span&gt;)&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="StageSpeedParam_2">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>310</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SetStageVelocity_2">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>310</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Set</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-160</x>
      <y>-40</y>
      <width>921</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/new/prefix1/desimoneresearchgroup.PNG&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>190</y>
      <width>141</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Parameter 
Selection</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>ExposureTime</zorder>
   <zorder>SliceThickness</zorder>
   <zorder>UVIntensity</zorder>
   <zorder>SelectResin</zorder>
   <zorder>SetExposureTime</zorder>
   <zorder>SetSliceThickness</zorder>
   <zorder>SetUVIntensity</zorder>
   <zorder>Title</zorder>
   <zorder>ResinSelect</zorder>
   <zorder>ResinSettlingTime</zorder>
   <zorder>ExposureTimeParam</zorder>
   <zorder>SliceThicknessParam</zorder>
   <zorder>ResinSettlingTimeParam</zorder>
   <zorder>SetResinSettlingTime</zorder>
   <zorder>FileList</zorder>
   <zorder>SelectFile</zorder>
   <zorder>ObjectImageFiles</zorder>
   <zorder>OutputScroll</zorder>
   <zorder>ProgramOutput</zorder>
   <zorder>InitializeAndSynchronize</zorder>
   <zorder>ProgramOutputLabel</zorder>
   <zorder>StartPrint</zorder>
   <zorder>StageVelocity</zorder>
   <zorder>UVIntensityParam</zorder>
   <zorder>StageSpeedParam</zorder>
   <zorder>SetStageVelocity</zorder>
   <zorder>MinEndOfRun</zorder>
   <zorder>MaxEndOfRun_2</zorder>
   <zorder>MinEndOfRunParam</zorder>
   <zorder>SetMinEndOfRun</zorder>
   <zorder>MaxEndOfRun</zorder>
   <zorder>SetMaxEndOfRun</zorder>
   <zorder>ManualLightEngine</zorder>
   <zorder>ManualStage</zorder>
   <zorder>CurrentStagePos</zorder>
   <zorder>CurrentStagePosition</zorder>
   <zorder>ViewGraphics</zorder>
   <zorder>CurrentPositionIndicator</zorder>
   <zorder>StartPrint_2</zorder>
   <zorder>StageVelocity_2</zorder>
   <zorder>StageSpeedParam_2</zorder>
   <zorder>SetStageVelocity_2</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>750</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
